monai/apps/auto3dseg/ensemble_builder.py

```diff
@@ -146,6 +165,8 @@ def __call__(self, pred_param: dict | None = None) -> list:
                       Example: `{"_target_": "SaveImage", "output_dir": "./"}`
                     - ``"sigmoid"``: use the sigmoid function (e.g. x > 0.5) to convert the prediction probability map
                       to the label class prediction, otherwise argmax(x) is used.
+                    - ``"model_specific_param"``: a dictionary to add pred_params that are specific to a model.
```

Comment:
I think the argument could be algo_params or algo_pred_params, according to the docstring

Reply:
@wyli How about `algo_spec_params` or `algo_specific` ? 
"algo_params" may sound a bit general, and since we may also add such param for training in AutoRunner as well, we would better not having "pred" in the name.

Reply:
Made the proposed edit in [9f53fdb](https://github.com/Project-MONAI/MONAI/pull/6487/commits/9f53fdbc501ecc7c3d6215417bad74895d316544), and please feel free to suggest other names.